app/build.gradle

```diff
@@ -185,6 +190,10 @@ dependencies {
     compile "com.google.firebase:firebase-database:${firebaseVersion}"
     compile "com.google.android.gms:play-services-auth:${firebaseVersion}"
 
+    def androidArch = '1.0.0-alpha6'
```

Comment:
Guys from the community are already using and some of them already have apps in production with alpha4.

If we had more complicated things, like in DeskDrop, I would wait. But we have a simple table and Room saves a lot of code. If a serious problem arises we can write the SQL code very fast.